api-specificatie/openapi.yaml

```diff
@@ -348,13 +348,20 @@ components:
       properties:
         nummeraanduidingIdentificatie:
           $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/lvbag/imbag/v20180601/nen3610.yaml#/NEN3610IDNum'
-        straatnaam:
+        verkorteStraatnaam:
           title: openbareruimte naam
           description: De verkorte versie van een naam die aan een openbare ruimte is toegekend in een daartoe strekkend formeel gemeentelijk besluit.
           type: string
           minLength: 1
-          maxLength: 80
+          maxLength: 24
           example: 'Laan v Westenenk'
+        straatnaam:
+          title: straatnaam
+          description: De volledige versie van een naam die aan een openbare ruimte is toegekend in een daartoe strekkend formeel gemeentelijk besluit.
+          type: string
+          minLength: 1
+          maxLength: 80
+          example: 'Laan van Westenenk'
```

Comment:
Zie mijn opmerking op de vorige pull request. Daar staat een beter voorbeeld in van een lange straatnaam die verkort wordt. Dit voorbeeld is niet correct omdat de lengte van de openbare ruimte naam 'Laan van Westenenk' < 24 karakters zal deze niet verkort worden.

Reply:
OK